python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -1472,6 +1471,10 @@ def submit(self, keep_going=None, keep_on_fail=None, prepare=False):
             # split into subjobs
             rjobs = self._doSplitting()
 
+            # Some old jobs may still contain None assigned to the self.subjobs so be careful when checking length
+            if self.subjobs is not None and len(self.subjobs) == 0 and self.splitter is not None:
```

Comment:
There are still potentially jobs before the default was changed. Minor number of jobs from minor number of users but I'm erring on the side of caution.